_cffi_build/build.py

```diff
@@ -36,14 +34,13 @@ def _mk_ffi(sources, name="_libevmjit", bundled=True, **kwargs):
 _base = [Source(absolute("evm.h"), "#include \"_cffi_build/evm.h\"",)]
 ffi = _mk_ffi(_base, libraries=['evmjit'])
 ffi.cdef("""
-    extern "Python" void  evm_query(struct evm_env* env,
+    extern "Python" union evm_variant evm_query(struct evm_env* env,
                                           enum evm_query_key key,
-                                          union evm_variant* arg,
-                                          union evm_variant* ret);
+                                          union evm_variant arg);
```

Comment:
I have not found anything saying that it will not work.

And I found this:
> CFFI supports passing and returning structs to functions and callbacks. 
> (...)
> There are a few (obscure) limitations to the argument types and return type. You cannot pass directly as argument a  union (but a pointer to a union is fine), nor a struct which uses bitfields (but a pointer to such a struct is fine). If you pass a struct (not a pointer to a struct), the struct type cannot have been declared with “...;” in the cdef(); you need to declare it completely in cdef(). You can work around these limitations by writing a C function with a simpler signature in the C header code passed to ffibuilder.set_source(), and have this C function call the real one.

> Aside from these limitations, functions and callbacks can receive and return structs.